tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -138,6 +238,36 @@ iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_waterm
     conditions:
       - "platform in ['x86_64-cel_e1031-r0']"
 
+#######################################
+#####            ip               #####
+#######################################
+ip/test_ip_packet.py:
+  skip:
+    reason: "Skipping ip packet test since can't provide enough interfaces"
+    conditions:
+      - "len(minigraph_interfaces) < 2 and len(minigraph_portchannels) < 2"
+
+#######################################
+#####            ipfwd            #####
+#######################################
+ipfwd/test_dip_sip.py:
+  skip:
+    reason: "Unsupported topology."
+    conditions:
+      - "topo_type not in ['t0', 't1', 't2']"
+
+ipfwd/test_dir_bcast.py:
+  skip:
+    reason: "Unsupported topology."
+    conditions:
+      - "topo_name not in ['t0', 't0-16', 't0-52', 't0-56', 't0-64', 't0-64-32', 't0-116']"
```

Comment:
It sounds like all `t0` type topologies are not supported, can you converge them to a condition like `"topo_type not in ['t0']"`